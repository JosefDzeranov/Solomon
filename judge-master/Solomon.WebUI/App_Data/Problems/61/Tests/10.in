65 58
ammo
shoot
ammo
ammo
ammo
shoot
shoot
shoot
shoot
shoot
ammo
ammo
shoot
shoot
shoot
ammo
ammo
ammo
ammo
shoot
shoot
ammo
shoot
ammo
ammo
ammo
ammo
ammo
shoot
shoot
shoot
shoot
ammo
ammo
ammo
ammo
ammo
shoot
ammo
shoot
ammo
shoot
shoot
shoot
ammo
ammo
shoot
shoot
shoot
shoot
shoot
ammo
ammo
shoot
shoot
shoot
ammo
ammo
shoot
shoot
shoot
ammo
ammo
ammo
ammo
