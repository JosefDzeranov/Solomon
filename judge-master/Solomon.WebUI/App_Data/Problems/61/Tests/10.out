58
57
115
173
231
230
229
228
227
226
284
342
341
340
339
397
455
513
571
570
569
627
626
684
742
800
858
916
915
914
913
912
970
1028
1086
1144
1202
1201
1259
1258
1316
1315
1314
1313
1371
1429
1428
1427
1426
1425
1424
1482
1540
1539
1538
1537
1595
1653
1652
1651
1650
1708
1766
1824
1882
